ONSdigital/response-operations-ui
# RAS-815: EQ_AND_SEFT when adding a eQ instruments SEFT ones will be removed

# What and why?
It has been found that when a user adds a SEFT CI first and then an eQ CI for a CE for a survey with a eq SEFT type, the SEFT will be removed and the user has to go back to re-add it before moving forward.

This fix will grab the CI when the eQ is added/removed and ensure that it is not removed when the eQ CI is updated.

# How to test?
1. Create a joint EQ SEFT survey (if one does not already exist)
2. Create a CE for that survey
3. Upload a SEFT CCI first
4. The add an eQ CI
5. When clicking 'Done' on the eq CI page, you will be returned to the CE overview page and the totals for the SEFT CI should remain unchanged
6. Click on the 'View or Upload' for the SEFT CI and the uploaded SEFTs should still be there.

As usual run the unit and acceptance tests.

#Jira
https://jira.ons.gov.uk/browse/RAS-815
